packages/p2p/src/components/orders/chat/chat-footer.jsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Input } from '@deriv/components';
+import PropTypes from 'prop-types';
+import { observer } from 'mobx-react-lite';
+import { localize } from 'Components/i18next';
+import ChatFooterIcon from 'Components/orders/chat/chat-footer-icon.jsx';
+import { useStores } from 'Stores';
+import ChatMessage from 'Utils/chat-message';
+
+const ChatFooter = observer(() => {
+    const { sendbird_store } = useStores();
+    const file_input_ref = React.useRef(null);
+    const text_input_ref = React.useRef(null);
+    const [character_count, setCharacterCount] = React.useState(0);
+
+    const updateTextAreaBounds = () => {
```

Comment:
Why not add these functions to a store?

Reply:
The functions are only useful for this particular component, there is no need to put them into a store.